src/assets/i18n/en.json

```diff
@@ -0,0 +1,327 @@
+{
```

Comment:
partially out-of-date with recent UI wording changes, potentially outdated unused keys, will be fixed in https://github.com/sovity/edc-ui/issues/828